tests/service/autoconfig/ispdbtest.php

```diff
@@ -50,12 +49,37 @@ public function queryData() {
 	 *
 	 * @param string $domain
 	 */
-	public function testQueryGmail($domain) {
-		$result = $this->ispDb->query($domain);
-
+	public function testQueryRealServers($domain) {
+		$ispDb = new IspDb($this->logger, Application::$ispUrls);
+		$result = $ispDb->query($domain);
 		$this->assertContainsIspData($result);
 	}
 
+	public function fakeAutoconfigData() {
+		return [
+			['freenet.de', true],
+			//['example.com', false], //should it fail?
```

Comment:
that does *not* fail, as we're not checking the ``emailProvider``. Is it redundant to check that?